library/common/extensions/filters/http/test_event_tracker/config.cc

```diff
@@ -0,0 +1,30 @@
+#include "library/common/extensions/filters/http/test_event_tracker/config.h"
+
+#include "library/common/extensions/filters/http/test_event_tracker/filter.h"
```

Comment:
Obviously, you're following an existing pattern (established by the assertion filter, among other things), but I'd strongly prefer we not be building test infrastructure into the core library. Not going to ask you to solve this in this PR (though if you did, that would be great), but did want to mention I don't think this is a long-term tenable approach.

Reply:
Discussed offline and created a GH issue to track this work https://github.com/envoyproxy/envoy-mobile/issues/1610